softioc/__init__.py

```diff
@@ -5,6 +5,10 @@
 from epicscorelibs.ioc import \
     iocshRegisterCommon, registerRecordDeviceDriver, pdbbase
 
+# Do this as early as possible, in case we happen to use cothread
+# as it's harmless if we don't
+import epicscorelibs.path.cothread  # noqa
```

Comment:
All this does is declare a single environment variable, so is harmless if `cothread` isn't installed

Reply:
But what is `epicscorelibs.path`?

I ended up navigating to [here](https://github.com/mdavidsaver/epicscorelibs/blob/master/src/python/epicscorelibs/path/cothread.py) to answer this question.  Ah well, a bit cryptic, but now I see.

Reply:
Maybe I need to put a better comment in
```suggestion
# This will set the CATOOLS_LIBCA_PATH environment variable in case we use cothread.catools
import epicscorelibs.path.cothread  # noqa
```